### PR TITLE
create home fragment

### PR DESCRIPTION
**Pull request by arsenkamavosyan**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 <resources>
     <string name="app_name">Daily aims</string>
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="home">Home</string>
+    <string name="statistics">Statistics</string>
+    <string name="edite">Edite</string>
```
Comment (artlazz) [inline app/src/main/res/values/strings.xml:6]: remove last 'e' from 'Edite'

File: app/src/main/res/menu/bottom_navigation_menu.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:id="@+id/page_1"
```
Comment (artlazz) [inline app/src/main/res/menu/bottom_navigation_menu.xml:4]: change page names to real fragment's names

File: app/src/main/res/menu/bottom_navigation_menu.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        android:enabled="true"
+        android:icon="@drawable/ic_baseline_home_24"
+        android:iconTint="@color/yellow"
+        android:title="@string/home"/>
+    <item
+        android:id="@+id/page_2"
```
Comment (artlazz) [inline app/src/main/res/menu/bottom_navigation_menu.xml:10]: same here

File: app/src/main/res/layout/card.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.cardview.widget.CardView
```
Comment (artlazz) [inline app/src/main/res/layout/card.xml:3]: rename this file
card.xml is not good name

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (arsenkamavosyan) [reply app/src/main/res/values/strings.xml:6]: uxxel em

File: app/src/main/res/menu/bottom_navigation_menu.xml (hunk 2 above)
Comment (arsenkamavosyan) [reply app/src/main/res/menu/bottom_navigation_menu.xml:4]: arden

File: app/src/main/java/com/dailyaims/screens/splash/home/HomeFragment.kt [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.dailyaims.Aims
+import com.dailyaims.databinding.FragmentHomeBinding
+import com.dailyaims.screens.splash.recycler_view_adapter.AimsAdapter
+
+class HomeFragment : Fragment() {
```
Comment (artlazz) [inline app/src/main/java/com/dailyaims/screens/splash/home/HomeFragment.kt:13]: when you create a fragment, create viewModel too, and try to replace 'logic' from fragment to viewModel

Comment (artlazz) [reply app/src/main/java/com/dailyaims/screens/splash/home/HomeFragment.kt:13]: in our case, get "Aims" list from ViewModel and when click on FAB tell viewModel to add new "Aim" to list

File: app/src/main/res/menu/bottom_navigation_menu.xml (hunk 3 above)
Comment (arsenkamavosyan) [reply app/src/main/res/menu/bottom_navigation_menu.xml:10]: ok

File: app/src/main/res/layout/card.xml (hunk 4 above)
Comment (arsenkamavosyan) [reply app/src/main/res/layout/card.xml:3]: inch dnem ba anun@

**Pull request merged**